templates/prometheus.go

```diff
@@ -45,25 +52,50 @@ var PrometheusTemplate = promv1.Prometheus{
 				Port:      intstr.FromString("web"),
 			}},
 		},
-		Resources: utils.GetResourceRequirements("prometheus"),
-		// prometheus tls -- will be uncommented
-		// Web: &promv1.WebSpec{
-		// 	TLSConfig: &promv1.WebTLSConfig{
-		// 		KeySecret: corev1.SecretKeySelector{
-		// 			Key: "tls.key",
-		// 		},
-		// 		Cert: promv1.SecretOrConfigMap{
-		// 			Secret: &corev1.SecretKeySelector{
-		// 				Key: "tls.crt",
-		// 			},
-		// 		},
-		// 		ClientAuthType: fmt.Sprint(tls.VerifyClientCertIfGiven),
-		// 		ClientCA: promv1.SecretOrConfigMap{
-		// 			ConfigMap: &corev1.ConfigMapKeySelector{
-		// 				Key: "service-ca.crt",
-		// 			},
-		// 		},
-		// 	},
-		// },
+		Resources:   utils.GetResourceRequirements("prometheus"),
+		ListenLocal: true,
+		Containers: []corev1.Container{{
+			Name:  "kube-rbac-proxy",
+			Image: "quay.io/brancz/kube-rbac-proxy:v0.12.0",
```

Comment:
- maybe use an env value to get this image? we can swap this with https://catalog.redhat.com/software/containers/openshift4/ose-kube-rbac-proxy/5cdb2634dd19c778293b4d98 when needed via env

Reply:
Yes, I just didn't want to commit downstream image to upstream repo. But using the env variable will solve the issue.